### PR TITLE
Add more description for console package

### PR DESCRIPTION
**Pull request by andybclee**
## Proposed changes

- This PR adds more description for console package

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: console/doc.go [hunk 1]
```diff
@@ -14,5 +14,18 @@
...
+
+Console is a JavaScript interpreted runtime environment. It is a fully fledged
+JavaScript console attached to a running node via an external or in-process RPC
+client.
+
+Source Files
+
+Each file provides following features
+ - bridge.go	: bridge is a collection of Javascript utility methods to bridge the .js runtime environment and the Go RPC connection backing the remote method calls
+ - console.go	: Implements console which support Javascript runtime environment
```
Comment (ehnuje) [inline console/doc.go]:
```suggestion
 - console.go	: Implements console which supports Javascript runtime environment
```

File: console/doc.go [hunk 2]
```diff
@@ -14,5 +14,18 @@
...
+ - prompter.go	: Provides UserPrompter which defines the methods needed by the console to promt the user for various types of inputs
```
Comment (ehnuje) [inline console/doc.go]:
```suggestion
 - prompter.go	: Provides UserPrompter which defines the methods needed by the console to prompt the user for various types of inputs
```

"the console to prompt the user for various types of inputs" sounds a bit confusing. Could you please refine this a bit?

File: console/doc.go (hunk 1 above)
Comment (KimKyungup) [inline console/doc.go]:
```suggestion
 - console.go	: Implements a console which supports Javascript runtime environment
```

File: console/doc.go [hunk 3]
```diff
@@ -14,5 +14,18 @@
...
+Package console implements Javascript console.
+
+Console is a JavaScript interpreted runtime environment. It is a fully fledged
+JavaScript console attached to a running node via an external or in-process RPC
+client.
+
+Source Files
+
+Each file provides following features
+ - bridge.go	: bridge is a collection of Javascript utility methods to bridge the .js runtime environment and the Go RPC connection backing the remote method calls
```
Comment (jeonghonah) [inline console/doc.go]:
```suggestion
 - bridge.go	: bridge is a collection of JavaScript utility methods to bridge the .js runtime environment and the Go RPC connection backing the remote method calls
```

File: console/doc.go (hunk 1 above)
Comment (jeonghonah) [inline console/doc.go]:
```suggestion
 - console.go	: Implements console which support JavaScript runtime environment
```

**Comment by andybclee** [comment]
@ehnuje @aidan-kwon Replaced remaining `Javascript` with `JavaScript`. Please take another look.

**Pull request merged**